packages/typespec-python/test/azure/generated/encode-bytes/encode/bytes/aio/operations/_operations.py

```diff
@@ -1496,22 +1496,19 @@ async def octet_stream(self, **kwargs: Any) -> bytes:
             map_error(status_code=response.status_code, response=response, error_map=error_map)
             raise HttpResponseError(response=response)
 
-        if _stream:
-            deserialized = response.iter_bytes()
-        else:
-            deserialized = await response.read()
+        deserialized = response.iter_bytes()
```

Comment:
This would still work even if `_stream` were `False` right? Only difference is that all the bytes would already be in memory?

Reply:
so this rolls back the ability to set `_stream` to `False`. We didn't document the `stream` kwarg here to begin with, and idt we should start out with support for passing `stream=False`

Reply:
Currently azure-core will [report error](https://github.com/Azure/azure-sdk-for-python/blob/4207b9f110dceb06adf7cd285e5c264ff2245a33/sdk/core/azure-core/azure/core/pipeline/policies/_universal.py#L646) for raw binary response when `stream=False` so actually SDK only support `stream=True` for now. So I am OK for the change. Once azure-core supports it, maybe we could add the code back.